fix(image): introduce a new setting `start-pre-pull-manager-images` (#922)
longhorn/website: content/docs/1.7.0/references/settings.md

The value in seconds specifies the timeout of the HTTP client to the replica's file sync server used for replica rebuilding, volume cloning, snapshot cloning, etc.

#### Pre-Pull Manager Images

> Default: `true`

> The setting enables pre-pulling Longhorn managers' images on all worker nodes before starting the managers' pods.

Reviewer: ```suggestion
Setting that allows worker nodes to pull Longhorn Manager images before Longhorn Manager pods are started.
```
